src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  transition: all 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: 0.3s;
+  }
+
+  &__img {
+    margin-bottom: 40px;
+    min-height: 134px;
+    padding-inline: 3px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 0;
+    margin-bottom: 4px;
+    color: #060b35;
+
+    &:hover .cart__title {
+      color: #34568b;
+      transition: 0, 3s;
```

Comment:
There is a typo in the transition property. It should be `transition: 0.3s;` instead of `transition: 0, 3s;`. This will ensure the hover effect on the `&__title` works smoothly.